spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -213,6 +213,7 @@ object SpdxDeclaredLicenseMapping {
         "cc by-sa 4.0" to CC_BY_SA_4_0.toExpression(),
         "cpal 1.0" to CPAL_1_0.toExpression(),
         "cpal v1.0" to CPAL_1_0.toExpression(),
+        "CUP Parser Generator Copyright Notice, License, and Disclaimer" to HPND.toExpression(),
```

Comment:
Commit message nits:
* Putting long links at the end improves readability, e.g. "As found in the POM file [1], ..."
* "Whereas" indicates a contrast between two facts, you should just use "where" instead.

Reply:
Done.